### ISSUE TITLE
Please add options to disable azote, and to disable or enable the hyprland splash text rendering

### ISSUE THREAD

**Issue by ethereal-engineer** [nwg-piotr/nwg-shell#293]
With the latest update of hyprland, the backgrounds have gone even beyond hyprchan (whom I loved anyway), and it would be a pity not to allow an option to see them when using nwg-shell. As for the splash text rendering, I really don't like having that but hyprland allows it to be turned off by a setting in misc. I already have my own sources doing this but I thought to mention it here for completeness.

**Comment by ethereal-engineer** [comment]
If you'd prefer these kind of thoughts in discussions instead, I apologise. I'm of limited ability rn.

**Comment by nwg-piotr** [comment]
If it comes to the `exec-once = ~/.azotebg-hyprland` line, I decided to leave it in the main config file on the very beginning. Surely I could make it switchable by adding a check button in the shell config utility and exporting this line to the `includes` file, but I have no way to remove it from all the current users' main config file. That's why I'd rather not do it.

If it comes to the Hyprland logo in the background, you have the switch in 'Miscellaneous' settings.

![obraz](https://github.com/nwg-piotr/nwg-shell/assets/20579136/dea4f5b5-8771-4ba3-b389-60127d886fa8)

**Comment by ethereal-engineer** [comment]
That's odd. I am pretty sure the azote line lives in the includes file. I will check shortly. As for the splash text, this is seperate to the hyprland logo. It is the text rendered at the bottom of the screen only. It has its own option to disable. I will report back on both soon.

**Comment by ethereal-engineer** [comment]
You're right. Azote is part of the pre-defined hyprland config file for nwg-shell. The other matter, the option for toggling the randomised splash text is here:

```
misc {
    disable_splash_rendering = true
}
```

**Comment by nwg-piotr** [comment]
I'll add this to nwg-shell-config, if so.
